bin/check-rds-pending.rb

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env ruby
+#
+# check-rds-events
+#
+#
+# DESCRIPTION:
+#   This plugin checks rds clusters for critical events.
+#   Due to the number of events types on RDS clusters the check searches for
+#   events containing the text string 'has started' or 'is being'.  These events all have
+#   accompanying completiion events and are impacting events
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk-v1
+#   gem: sensu-plugin
+#
+# USAGE:
+#  ./check-rds-events.rb -r ${you_region} -s ${your_aws_secret_access_key} -a ${your_aws_access_key}
+#
+# NOTES:
+#
+# LICENSE:
+#   Tim Smith <tim@cozy.co>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckRDSEvents < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
```

Comment:
remove both the aws_access_key and aws_secret_access_key as these should not be passed in on the command line.